app/containers/App/Application.js

```diff
@@ -152,6 +153,8 @@ class Application extends React.Component {
           <Route path="/admin/edit/:id/teams" component={TeamsPage} />
           <Route path="/admin/edit/:id/vendors" component={VendorsPage} />
           <Route path="/admin/edit/:id/testimonials" component={TestimonialsPage} />
+          <Route path="/admin/edit/:id/registration" component={RegisterPage} />
```

Comment:
i'm surprised this was not here before.  What is the intent behind adding it if it was not here already?

Reply:
This call was missing - when I tested the ability to customize the registration page I forgot to commit this file.